### ISSUE TITLE
[Feature Request] Show plan output even when force=true

### ISSUE THREAD

**Issue by jeffgran** [minamijoyo/tfmigrate#39]
Maybe this should be an optional flag, if some prefer it the way it is now. 

But as I have been using `tfmigrate`, I often expect a few specific diffs to be in the final plan, but I still want to verify that they look correct, even when I use `force = true` to apply them anyway.

Currently, my workflow is to comment out the `force = true` line, run the `plan`, verify it, then turn uncomment the line to run the `apply`. It works but I often forget and have to run things over again!  

--

Also thank you for writing this tool, it is amazing and has made my life so much easier! Another feature request is a donate button somewhere so I can show some monetary support :)

**Comment by minamijoyo** [comment]
I think the feature was implemented in v0.2.2 via https://github.com/minamijoyo/tfmigrate/pull/11.
I cannot reproduce it with the latest (v0.2.7). Which version are you using?

```
bash-5.0# tfmigrate -v
0.2.7

bash-5.0# terraform state list
aws_security_group.bar
aws_security_group.foo

bash-5.0# cat main.tf
resource "aws_security_group" "foo2" {
  name = "foo"
}

resource "aws_security_group" "bar2" {
  name = "bar"
}

bash-5.0# cat test.hcl
migration "state" "test" {
  force = true
  actions = [
    "mv aws_security_group.foo aws_security_group.foo2",
  ]
}

bash-5.0# tfmigrate plan test.hcl
2021/09/03 07:15:37 [INFO] [runner] load migration file: test.hcl
2021/09/03 07:15:37 [INFO] [migrator] start state migrator plan
2021/09/03 07:15:37 [INFO] [migrator@.] terraform version: 1.0.5
2021/09/03 07:15:37 [INFO] [migrator@.] initialize work dir
2021/09/03 07:15:39 [INFO] [migrator@.] switch to remote workspace default
2021/09/03 07:15:40 [INFO] [migrator@.] get the current remote state
2021/09/03 07:15:41 [INFO] [migrator@.] override backend to local
2021/09/03 07:15:41 [INFO] [executor@.] create an override file
2021/09/03 07:15:41 [INFO] [migrator@.] creating local workspace folder in: terraform.tfstate.d/default
2021/09/03 07:15:41 [INFO] [executor@.] switch backend to local
2021/09/03 07:15:43 [INFO] [migrator@.] compute a new state
2021/09/03 07:15:43 [INFO] [migrator@.] check diffs
2021/09/03 07:15:46 [INFO] [migrator@.] unexpected diffs, ignoring as force option is true: failed to run command (exited 2): terraform plan -state=/tmp/tmp096906093 -out=/tmp/tfplan045121000 -input=false -no-color -detailed-exitcode
stdout:
aws_security_group.bar: Refreshing state... [id=sg-e5de8a85]
aws_security_group.foo2: Refreshing state... [id=sg-21ac42f1]

Note: Objects have changed outside of Terraform

Terraform detected the following changes made outside of Terraform since the
last "terraform apply":

  # aws_security_group.foo2 has been changed
  ~ resource "aws_security_group" "foo2" {
        id                     = "sg-21ac42f1"
        name                   = "foo"
      + tags                   = {}
        # (7 unchanged attributes hidden)
    }
  # aws_security_group.bar has been changed
  ~ resource "aws_security_group" "bar" {
        id                     = "sg-e5de8a85"
        name                   = "bar"
      + tags                   = {}
        # (7 unchanged attributes hidden)
    }

Unless you have made equivalent changes to your configuration, or ignored the
relevant attributes using ignore_changes, the following plan may include
actions to undo or respond to these changes.

─────────────────────────────────────────────────────────────────────────────

Terraform used the selected providers to generate the following execution
plan. Resource actions are indicated with the following symbols:
  + create
  - destroy

Terraform will perform the following actions:

  # aws_security_group.bar will be destroyed
  - resource "aws_security_group" "bar" {
      - arn                    = "arn:aws:ec2:ap-northeast-1:000000000000:security-group/sg-e5de8a85" -> null
      - description            = "Managed by Terraform" -> null
      - egress                 = [
          - {
              - cidr_blocks      = [
                  - "0.0.0.0/0",
                ]
              - description      = ""
              - from_port        = 0
              - ipv6_cidr_blocks = []
              - prefix_list_ids  = []
              - protocol         = "-1"
              - security_groups  = []
              - self             = false
              - to_port          = 0
            },
        ] -> null
      - id                     = "sg-e5de8a85" -> null
      - ingress                = [] -> null
      - name                   = "bar" -> null
      - owner_id               = "000000000000" -> null
      - revoke_rules_on_delete = false -> null
      - tags                   = {} -> null
      - tags_all               = {} -> null
    }

  # aws_security_group.bar2 will be created
  + resource "aws_security_group" "bar2" {
      + arn                    = (known after apply)
      + description            = "Managed by Terraform"
      + egress                 = (known after apply)
      + id                     = (known after apply)
      + ingress                = (known after apply)
      + name                   = "bar"
      + name_prefix            = (known after apply)
      + owner_id               = (known after apply)
      + revoke_rules_on_delete = false
      + tags_all               = (known after apply)
      + vpc_id                 = (known after apply)
    }

Plan: 1 to add, 0 to change, 1 to destroy.

─────────────────────────────────────────────────────────────────────────────

Saved the plan to: /tmp/tfplan045121000

To perform exactly these actions, run the following command to apply:
    terraform apply "/tmp/tfplan045121000"

stderr:
2021/09/03 07:15:46 [INFO] [executor@.] remove the override file
2021/09/03 07:15:46 [INFO] [executor@.] remove the workspace state folder
2021/09/03 07:15:46 [INFO] [executor@.] switch back to remote
2021/09/03 07:15:48 [INFO] [migrator] state migrator plan success!
```

---
P.S. Thank for your offering, but I'm currently not accepting any money for my OSS projects. Thanks!

**Comment by jeffgran** [comment]
Hi, thank you for your reply! I am on the latest version:

```
 $ tfmigrate -v
0.2.7
```

I think maybe this only happens when it's a `multi_state` migration. I should have mentioned that in the original post, sorry.

**Comment by minamijoyo** [comment]
@jeffgran I confirmed and fixed it in v0.2.9 via #40. Thank you for reporting!
